README.md

```diff
@@ -35,6 +35,22 @@ To use N-API in a native module:
 #include "napi.h"
 ```
 
+To make the migration to node-api easier, we have also provide a script to do these steps above for you. To use the conversion script:
+  1. Go to your module directory
+```
+cd [module_path]
+```
+  2. Install node-api module
+```
+npm install node-api
+```
+  3. Run node-api conversion script
+```
+node ./node_modules/node-api/tools/conversion.js ./
+```
+  4. This script is to make the migration easier and it is possible to miss out some conversion. So next step is trying to build your module and fix what the script is missing.
```

Comment:
How about we say "We tried our best to cover all common replacement patterns. This script should be able to handle majority of the migration work."